### DISCUSSION TITLE
lettuce version compatibility with redis-cluster

### DISCUSSION THREAD

**Discussion by Bharat-oyo** [redis/lettuce#2246] [General]
I'am currently using lettuce 5.0.4 with redis-cluster version 5.0 now I want to update redis-cluster version to 6.0, so I want to know that Is lettuce 5.0.4 is compatible with redis-cluster version to 6.0? How can I check this compatibility?

**Comment by mp911de** [comment]
Lettuce is compatible with Redis 2.6++ up to the latest Redis version. We're building against the `unstable` Redis branch to ensure compatibility.
